edb/lang/schema/objects.py

```diff
@@ -223,16 +223,17 @@ def __repr__(self):
 
 class SchemaField(Field):
 
-    __slots__ = ('default', 'hashable')
+    __slots__ = ('default', 'hashable', 'stdonly')
```

Comment:
We already have `public` to allow fields to be set from DDL.  We could allow setting **any** field when in stdmode, and then we don't need your flag.  IMHO that's a better solution than adding yet another flag.

Reply:
Sure.  This would require a separate commit to mark all the necessary fields as public, as we don't currently rely on it in DDL at all.

Reply:
Another difficulty here is that some fields are set programmatically by the DDL pipeline as a result of some syntax.  For example `Function.language` and `Function.code` are technically public, since you can set them with a DDL command, but then also `SET language :=` should be illegal.  Since we do the check in `AlterObjectProperty`, it makes it hard to distinguish between the two.

Reply:
OK, I renamed `public` to `allow_ddl_set` and switched everything to this whitelist in an additional commit.